tests/functionnal/Glpi/Inventory/Inventory.php

```diff
@@ -2570,7 +2502,7 @@ public function testImportStackedNetworkEquipment()
         $this->integer(count($db_unmanageds))->isIdenticalTo(45);
 
         $db_ips = $DB->request(['FROM' => \IPAddress::getTable()]);
-        $this->integer(count($db_ips))->isIdenticalTo(150);
+        $this->integer(count($db_ips))->isIdenticalTo(5);
```

Comment:
This change seems use, do you know why?

Reply:
Each management IP are duplicate 5 times.

```json
"ips": [
                "10.1.0.100",
                "10.1.0.22",
                "10.1.0.41",
                "10.1.0.45",
                "10.1.0.59",
                "10.11.11.1",
                "10.11.11.5",
                "10.11.13.1",
                "10.11.13.5",
                "172.21.0.1",
                "172.21.0.7",
                "172.22.0.1",
                "172.22.0.5",
                "172.23.0.1",
                "172.23.0.5",
                "172.24.0.1",
                "172.24.0.5",
                "172.25.1.15",
                "172.28.200.1",
                "172.28.200.5",
                "172.28.211.5",
                "172.28.215.1",
                "172.28.221.1",
                "185.10.253.65",
                "185.10.253.97",
                "185.10.254.1",
                "185.10.255.146",
                "185.10.255.224",
                "185.10.255.250"
            ],
``` 

because it's a stacked switch (5 stacks)

![image](https://user-images.githubusercontent.com/7335054/218764558-0f0270f2-2450-44af-8c4a-ec30fe88db60.png)

Each ```stack``` have a ```management port``` with all ```IPs```

![image](https://user-images.githubusercontent.com/7335054/218764714-3c148b21-d52c-4649-8686-6c85f8b3a79c.png)

But now, we only have one ```IP``` per management port (multiplied by the size of the stack) => 5